README.md

```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
 
 ## Installation
 
-**TODO** Update the sections below:
+1. cd into backend folder
+    cd backend
+
+2. create virtual environment
+    ### Installing virtualenv
+    ```pip install virtualenv```
+
+    ### Creating virtual environment
+    ```python -m venv venv```
+
+    ### Activating the virtual environment
+    ```source venv/bin/activate```
```

Comment:
This is good! Some of our users might be on Windows machines, so let's add instructions on how to activate the virtual environment on Windows. The command looks something like this:

```venv\Scripts\activate```

Reply:
Noted. Thanks Vuyie.